ms-frontend/src/services/reviewService.ts

```diff
@@ -1,22 +1,35 @@
 import apiClient from '@/services/axios'
 import { Review } from '@/types/Review'
 import { LocationReviews } from '@/types/LocationReviews'
+import { Location } from '@/types/Location'
 
 export const reviewService = {
+  async createReview(text: string, rating: number, location: Location) {
+    await apiClient.post('/reviews', {
+      text: text,
+      rating: rating,
+      location: {
+        id: location.id,
+        name: location.name,
+        type: location.type,
+        coordinates: {
+          x: location.coordinates.x,
+          y: location.coordinates.y
+        }
+      }
+    })
+  },
+  
```

Comment:
```suggestion
    async appendReviewImage(reviewId: string, file: File) {
    const formData = new FormData()
    formData.append('review_id', reviewId)
    formData.append('image', file)

    const response = await apiClient.patch('reviews/image', formData)
    return new Review(response.data)
  },
```

Add to append a review image.